City-of-Helsinki/helsinki-paatokset
# PP-202: Policymaker documents page

Create view to function as a documents list page for policymakers.

In this branch:

- Created new service to function as a data fetcher for `paatokset_submenus` -module's blocks.
- Moved minutes of discussion list to it's own block
- Created new block to list all documents related to policymaker (both minutes of discussion & declarations of affiliation)
- Created path alias pattern for policymakers

To test: 

- Run `drush cim`
- Go to `https://helsinki-paatokset.docker.so/fi/admin/config/search/path/update_bulk` and generate path aliases
- Create media content of type minutes of discussion or declaration of affiliation. Link created media to a policymaker / meeting
- View the policymakerpage.  Add `/asiakirjat` to the url (as in `paattajat/kaupunginvaltuusto` -> `paattajat/kaupunginvaltuusto/asiakirjat`.
- List should look like this:
![image](https://user-images.githubusercontent.com/49063836/127509509-73690345-1b29-47b7-8a2f-bedf30e623c4.png)
